registry/reference.go

```diff
@@ -42,7 +42,14 @@ var (
 	tagRegexp = regexp.MustCompile(`^[\w][\w.-]{0,127}$`)
 )
 
-// Reference references to a descriptor in the registry.
+// Reference references either a resource descriptor (where Reference.Reference
+// is a non-empty string), or a resource repository (where Reference.Reference
+// is the empty string.
```

Comment:
Can we put what in the brackets to the documentation of `Reference string` at line 63?